src/main/java/sonia/scm/issuetracker/internal/PullRequestCommentLinkEnricher.java

```diff
@@ -0,0 +1,99 @@
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package sonia.scm.issuetracker.internal;
+
+import com.cloudogu.scm.review.comment.service.Comment;
+import com.google.common.base.Strings;
+import sonia.scm.api.v2.resources.Enrich;
+import sonia.scm.api.v2.resources.HalAppender;
+import sonia.scm.api.v2.resources.HalEnricher;
+import sonia.scm.api.v2.resources.HalEnricherContext;
+import sonia.scm.issuetracker.IssueLinkFactory;
+import sonia.scm.issuetracker.IssueMatcher;
+import sonia.scm.issuetracker.IssueTracker;
+import sonia.scm.plugin.Extension;
+import sonia.scm.plugin.Requires;
+import sonia.scm.repository.Repository;
+
+import javax.inject.Inject;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@Extension
+@Enrich(Comment.class)
+@Requires("scm-review-plugin")
+public class PullRequestCommentLinkEnricher implements HalEnricher {
+
+  private final IssueTrackerManager issueTrackerManager;
+
+  @Inject
+  public PullRequestCommentLinkEnricher(IssueTrackerManager issueTrackerManager) {
+    this.issueTrackerManager = issueTrackerManager;
+  }
+
+  @Override
+  public void enrich(HalEnricherContext context, HalAppender appender) {
+
+    Repository repository = context.oneRequireByType(Repository.class);
+    Comment comment = context.oneRequireByType(Comment.class);
+
+    Map<String, String> map = createIssueLinkMap(repository, comment);
+
+    if (!map.isEmpty()) {
+      HalAppender.LinkArrayBuilder halArrayBuilder = appender.linkArrayBuilder("issues");
+      map.forEach(halArrayBuilder::append);
+      halArrayBuilder.build();
+    }
+  }
+
+  private Map<String, String> createIssueLinkMap(Repository repository, Comment comment) {
+    Map<String, String> map = new LinkedHashMap<>();
+
+    for (IssueTracker issueTracker : issueTrackerManager.getIssueTrackers()) {
+      Optional<IssueMatcher> issueMatcher = issueTracker.createMatcher(repository);
+
+      if (!issueMatcher.isPresent()) continue;
+
+      Pattern pattern = issueMatcher.get().getKeyPattern();
+
+      Matcher matcher = pattern.matcher(Strings.nullToEmpty(comment.getComment()));
+
+      Optional<IssueLinkFactory> issueLinkFactory = issueTracker.createLinkFactory(repository);
+
+      if (issueLinkFactory.isPresent()) {
+        while (matcher.find()) {
+          String key = issueMatcher.get().getKey(matcher);
+          String link = issueLinkFactory.get().createLink(key);
+          map.put(key, link);
+        }
+      }
+    }
+
+    return map;
+  }
```

Comment:
This looks like a exact copy from a method of `ChangesetLinkEnricher`. Please create a util and use it from both classes.